.github/pull-request-template.md

```diff
@@ -1,4 +1,11 @@
-Fixes #(issue-number)
+Fixes #issue-number
+
+<!-- Most PRs should be opened against the master branch.
+
+If the change should also be in the most recent release, add the
```

Comment:
```suggestion
If the change should also be in the most recent release (master branch), you must add the
```

Reply:
By the "most recent release," I meant the most recent numbered release, not the master branch.

Reply:
Gotcha, sorry, i see that now.

The "use case" in my mind was a non contributor (so read this too quickly and from a different perspective). For example, I was thinking that this should focus on helping someone who has not read the guidelines and is just trying to fix something in their release (most recent release).

Maybe we should reconsider the guidance around "open the PR against Master and the cherrypick it into a branched release"? If someones goal is to fix something in a release they are using, and they never get to cherrypicking a change from master into a branch-release, then the overhead to "contributing a quick fix" might be more than what "today's usual contributor" is (ie today we mostly get PRs from developers working on code for v.Next). 

Thoughts?

Reply:
I think the reasoning here is that, if you commit into a numbered release instead, and it's a change that should be carried forward, the user then needs to cherry-pick the change into master. In theory, most changes made probably should be carried forward, so I think we'd be doing a lot more cherry-picking overall if we directed people to make commits into a release branch, since not all changes need to be carried back into the last release.

I also think the current workflow is aligned with what the other repos in Knative do, and I get the feeling it's a pretty common Git workflow.